src/appmixer/twitter/bundle.json

```diff
@@ -1,12 +1,15 @@
 {
     "name": "appmixer.twitter",
-    "version": "2.0.2",
+    "version": "3.0.0",
```

Comment:
it's not a breaking change now, right?

Reply:
I think it shouldn't be 
 Only icons and activity descriptions and label of app was changed